### ISSUE TITLE
MA0137 probably should have an internal exclusion list

### ISSUE THREAD

**Issue by mjr4077au** [meziantou/Meziantou.Analyzer#1205]
### Version of the Meziantou.Analyzer NuGet package

3.0.115

### Rule Identifier

MA0137

### Target Framework

net472, net8.0

### C# Language version

C# 14

### Description

For this rule, I'd expect it to not fire on properties named `Task` or methods named `ConfigureAwait()` as both names are standardised naming patterns.

<img width="1082" height="211" alt="Image" src="https://github.com/user-attachments/assets/b20b71aa-7d02-4d21-9128-932475f14a86" />

### Reproduction Steps

Minimal code:

```c#
code to reproduce the error
```

**Comment by mjr4077au** [comment]
Confirming this one's also good now. Thanks again!
